### PR TITLE
Added recipe for interlap

### PR DESCRIPTION
**Pull request by jenzopr**
Hi all,
although Brent Pedersen examplifies use in the genomics context, I think the interlap package is general enough to be included in the conda-forge channel. That's why I PR here.

InterLap is >20 times faster than doing a naive search (see: https://brentp.github.io/interlap/benchmark.html) with no memory overhead because it operates on a sorted list. It is pure python and has no dependencies.

It uses binary search and a knowledge of the longest observed interval to quickly query datasets with 100's of thousands of intervals.

Best,
Jens

### CODE REVIEW

**Comment by jenzopr** [comment]
Unfortunately, the `LICENSE` file is not included in the Pypi source package. I'm now waiting for the upstream author to include it.

File: recipes/interlap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
```
Comment (synapticarbors) [inline recipes/interlap/meta.yaml]: Don't use jinja templating here. Just the explicit import.

Comment (jenzopr) [reply recipes/interlap/meta.yaml]: Thanks for the comment! Just for curiosity: Which advantage does the explicit import have?

**Comment by jenzopr** [comment]
@synapticarbors 
I changed to an explicit import during test, but it seems the travis build got stuck. Is there a way to trigger it again? 
Thanks a lot!

**Comment by synapticarbors** [comment]
@jenzopr -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**